tools/proteomiqon_psmbasedquantification/proteomiqon_psmbasedquantification.xml

```diff
@@ -0,0 +1,449 @@
+                                    ]
+                                }
+                            }
+                        ]    
+                        #end if
+                    },
+                    #if $XicExtraction.TopKPSMsCond.TopKPSMsEnabled == "yes"
+                    "TopKPSMs": {
+                        "Case": "Some",
+                        "Fields": [
+                            ${XicExtraction.TopKPSMsCond.TopKPSMs}
+                        ]
+                    }
+                    #else
+                    "TopKPSMs": null
+                    #end if
+                },
+                #if $BaseLineCorrectionCond.BaseLineCorrectionEnabled == "yes"
+                "BaseLineCorrection": {
+                    "Case": "Some",
+                    "Fields": [
+                        {
+                            "MaxIterations":$BaseLineCorrectionCond.MaxIterations,
+                            "Lambda":$BaseLineCorrectionCond.Lambda,
+                            "P":$BaseLineCorrectionCond.P
+                        }
+                    ]
+                }
+                #else
+                "BaseLineCorrection": null
+                #end if
+            }
+            ]]>
+        </configfile>
+    </configfiles>
+    <inputs>
+        <param name="instrumentOutput" type="data" format="sqlite" label="Instrument output" help="Specify mass spectrometry data you want to analyze."/>
+        <param name="scoredPSMs" type="data" format="tabular" label="Scored PSM file" help="Specify list of peptide identifications."/>
+        <param name="peptideDB" type="data" format="sqlite" label="Peptide database" help="Specify the peptide data base."/>
+        <param name="PerformLabeledQuantification" type="boolean" checked="true" label="Perform labeled quantification" help="If checked, a quantification of peptides which contain an isotopic modification is performed. Additionally, for every identified peptide we check if the mass spectrometry run contains a peak at the m/z of the labeled/unlabeled counterpart and quantify it."/>
+        <section name="XicExtraction" title="XIC extraction">
+            <param name="ScanTimeWindow" type="float" value="2.0" label="Scan timewindow" help="Specify the length of the scan time window used for XIC creation."/>
+            <conditional name="MzWindow_DaTypeCond">
+                <param name="MzWindow_DaType" type="select" label="MzWindow_Da type" help="Specify the MzWindow Da Type">
+                    <option value="Fixed" selected="true">Fixed</option>
+                    <option value="Estimate">Estimate</option>
+                </param>
+                <when value="Fixed">
+                    <param name="MzWindow_Da" type="float" value="0.07" label="Value" help="Specify the m/z window set centered on the target m/z used for XIC creation" />
```

Comment:
label value is a little bit strange :) Please explain which value

Reply:
Good point! Will change :)